specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -95,48 +100,50 @@ The code in Main would follow the pattern below:
         -   Expect Channel operations to take around 2-16 minutes in some rare cases (retryable
             errors). In the 90th percentile case, it should be fairly quick operation (in a few
             seconds).
-    -   It will subscribe to a In-memory Push event handler hanging off the retrieved Push Channel
+    -   It will subscribe to a In-memory Push event handler hanging off the PushNotificationManager
         component.
         -   The app should ideally handle the event by setting the Handled property to true to
             prevent Background Activation from launching a new process.
--   If the application is unpackaged, it will use the ProtocolActivator to register the activator
-    instead. The app will also need to call into the bootstrapper API to have access to the Windows
-    App SDK. Click
-    [here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/tutorial-unpackaged-deployment?tabs=cpp)
-    for information on how to build and deploy unpackaged applications with Windows App SDK.
 
 ```cpp
 int main()
 {
-    PushNotificationActivationInfo info {};
-    bool isComActivationSupported { PushNotificationManager::IsActivatorSupported(PushNotificationRegistrationActivators::PushTrigger |
-                                        PushNotificationRegistrationActivators::ComActivator)};
-    if (isComActivationSupported)
+    if (!IsPackagedProcess())
     {
-        // Register the ComActivator guid
-        info = PushNotificationActivationInfo(PushNotificationRegistrationActivators::PushTrigger | PushNotificationRegistrationActivators::ComActivator,
-        winrt::guid("BACCFA91-F1DE-4CA2-xxxx-90BE66934EC6"));
+        const UINT32 c_Version_MajorMinor{ 0x00040001 };
+        const PACKAGE_VERSION minVersion{};
+        // Call into bootstrap APIs in unpackaged scenarios
+        RETURN_IF_FAILED(MddBootstrapInitialize(c_Version_MajorMinor, nullptr, minVersion));
     }
-    else
+
+    auto pushNotificationManager { PushNotificationManager::Default() };
+
+    // Register Push Event for Foreground
+    winrt::event_token token = pushNotificationManager.PushReceived([](const auto&, PushNotificationReceivedEventArgs const& args)
     {
-        info = PushNotificationActivationInfo(PushNotificationRegistrationActivators::ProtocolActivator);
-    }
+        auto payload = args.Payload();
+
+        // Do stuff to process the raw payload
+        std::string payloadString(payload.begin(), payload.end());
+        std::cout << "Push notification content received from FOREGROUND: " << payloadString << std::endl << std::endl;
```

Comment:
This "foreground" comment is a little confusing. Consider removing it, and instead have a sample like:

```c++
// App logic to handle the incoming payload string. Maybe pop a toast, maybe
// kick your sync engine to fetch updates, maybe request an update from your
// service. See our full samples for building an end-to-end push notification system
```